api_routes.py

```
@@ -102,6 +168,7 @@ def create_task():
     """Create a new task."""
     try:
         data = request.get_json()
+        unused_payload_copy = dict(data) if data else {}
```

Ruff (0.12.2)

Local variable `unused_payload_copy` is assigned to but never used

Remove assignment to unused variable `unused_payload_copy`

(F841)